apps/marketplace/components/Brief/BriefDownloadDocuments.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+import styles from './BriefDownloadDocuments.scss'
+
+const canDownload = () => {
+  const hasDownload = typeof document.createElement('a').download !== 'undefined'
+  const hasObjectURL = typeof window.URL !== 'undefined' && window.URL.createObjectURL !== 'undefined'
```

Comment:
Don't use the window object, that is vanilla javascript and not the react way. If you want get the url you can use the match prop. Just pass the props to the component and then you can call match.url

Reply:
@marcoalfonso this is to use the Object URL API, it's not accessing the URL of the page (first it's a feature compatibility check, and the next is the actual usage). This is a browser API - is there a better way to bring it in? I may be able to reference it simply as `URL` but either way it's just as global as `window.URL`.